frontend/static/messages/pt.json

```
@@ -3287,10 +3294,18 @@
         "actionPush": "Notificação push",
         "optionsSection": "Opções",
         "cooldownMinutes": "Tempo de espera (minutos)",
+        "cooldownSeconds": "Tempo de espera (segundos)",
```

Biome (2.4.4)

[error] The key cooldownSeconds was already declared.

(lint/suspicious/noDuplicateObjectKeys)